src/App.js

```diff
@@ -23,6 +23,7 @@ import {
   faHome
 } from '@fortawesome/free-solid-svg-icons';
 import './App.css';
+import '../src/assets/css/potrait.css';
```

Comment:
potrait should be spelled "portrait"

Reply:
I kept dropping the r myself, apologies if you copied my typo... :)